tests/tests_ethernet.yml

```diff
@@ -1,20 +1,21 @@
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Please remove these changes from this PR.

Reply:
done